pkg/virt-config/config-map.go

```diff
@@ -228,17 +228,33 @@ func (c *ClusterConfig) SetConfigModifiedCallback(cb ConfigModifiedFn) {
 	go c.configModifiedCallback()
 }
 
+//
+type migrationConfiguration struct {
```

Comment:
can we document that this struct is being deprecated, and that no new values should be added moving forward. I just want to make sure we don't get into a situation where we have to continue to maintain two structs, one for the configmap and another for our kubevirt crd.

Reply:
Sure, do you know if we have already announced deprecation of config-map?

Reply:
Once your PR is merged and backported, we can merge the doc PR from @ashleyschuett and formally announce the deprecation.